CONTRIBUTING.md

```diff
@@ -128,6 +128,32 @@ cd ci
 summon ./test_e2e
 ```
 
+## Updating Dependencies
+
+1. Run `bundle install` command inside a Conjur Service Broker test container
+   to update Conjur Service Broker gems:
+
+   ```sh-session:q
+   docker-compose run tests bundle install
+   ```
+
+1. Test Conjur Service Broker functionality by starting up the development
+   environment as described in the
+   [Development Environment](#development-environment) and running the
+   local integration tests.
+
+1. The dependency update that is done via `bundle install` may be too
+   aggressive in the sense that it could introduce new dependencies with
+   which the Conjur Service Broker is incompatible. In this case, you may
+   need to examine the errors in the Cucumber test output, or in the
+   Conjur Service Broker container logs to determine which dependency version
+   needs to be dialed back. You may need to manually adjust dependency
+   versions in the `Gemfile.lock` file.
+
+1. After tweaking the `Gemfile.lock` file to get the integration tests to
```

Comment:
We should really avoid manually tweaking the `Gemfile.lock`. The best way to update dependencies is to use `bundle upgrade`. If we can't get that to work properly, then we can manually erase a line, then run `bundle upgrade` to fill it back in. Worst case scenario, just delete the `Gemfile.lock`, and run the tests, which will generate a new one.

Reply:
Yes, indeed! This section was re-written.